docs/classic-ui/page-templates/index.md

```diff
@@ -0,0 +1,196 @@
+# Page Templates in Plone Classic-UI
+
+These instructions and resources will help you develop Classic-UI pages.
+
+When you visit the Plone backend with a web browser, you will reach the Classic-UI.  These pages are generated with Classic Plone Browser Views and Page Templates.
+
+This chapter covers developing Page Templates.   Browser views are documented in-depth in (another chapter)
+# Contents
```

Comment:
Please insert 2 line breaks before headings, and 1 line break after.

Optionally add labels. In MyST syntax, we can use:

```
(dis-mah-heddin-label)=

## Dis Mah Heddin

Dis mah narrative text.
```

Also this should be a second-level heading, as the first level is the page title.